pkg/admission/customresourcevalidation/attributes.go

```diff
@@ -0,0 +1,46 @@
+package customresourcevalidation
+
+import (
+	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/apiserver/pkg/admission"
+)
+
+// unstructuredUnpackingAttributes tries to convert to a real object in the config scheme
+type unstructuredUnpackingAttributes struct {
```

Comment:
why this wrapper struct? An explicit conversion in the admission handler would be much simpler. No need for on-the-fly conversion.

Reply:
> why this wrapper struct? An explicit conversion in the admission handler would be much simpler. No need for on-the-fly conversion.

We don't know whether or not we can until point of use.  Also, point of use makes sure that that we don't miss a spot.